src/session/scylla_session.rs

```diff
@@ -83,4 +85,17 @@ impl ScyllaSession {
 
     Ok(result)
   }
+
+  #[napi]
+  pub async fn query(
+    &self,
+    scylla_query: &ScyllaQuery,
+    parameters: Option<Vec<Either3<u32, String, &Uuid>>>,
+  ) -> napi::Result<serde_json::Value> {
+    let values = QueryParameter::parser(parameters).unwrap();
+
+    let query_result = self.session.query(scylla_query.query.clone(), values).await;
```

Comment:
Do we need to clone the `scylla_query` here? We already have a reference of it and is no longer used, at least here